options/locale/locale_en-US.ini

```diff
@@ -940,7 +940,8 @@ issues.new.no_assignees = No Assignees
 issues.new.no_reviewers = No reviewers
 issues.new.add_reviewer_title = Request review
 issues.choose.get_started = Get Started
-issues.choose.blank = Open a blank issue
+issues.choose.blank = Blank issue
+issues.choose.blank_about = Create an issue without a template. 
```

Comment:
There may or may not still be an issue at play, as default will still look for the legacy template.